conf/config.toml

```diff
@@ -52,15 +52,18 @@ region-schedule-limit = 16
 replica-schedule-limit = 24
 
 # customized schedulers, the format is as below
-# if empty, it will use balanceLeader, balanceRegion, hotReadRegion, hotWriteRegion as default
-[schedule.schedulers]
-# [schedule.schedulers.balanceLeader]
+# if empty, it will use balance-leader, balance-region, hot-region as default
```

Comment:
please use two PRs, one is to change the name, the other is to persist the config.